### PR TITLE
[DEV-587] min deposit changes

### PR DESCRIPTION
**Pull request by Krysh90**
depends on
- https://github.com/DFXswiss/payment/pull/142 being merged first, due to all changes made to `RouteList`
- https://github.com/DFXswiss/api/pull/350

### CODE REVIEW

File: app/screens/home/RouteList.tsx [hunk 1]
```diff
@@ -252,7 +253,11 @@ const RouteList = ({
       onPress: () => ClipboardService.copy(route.deposit?.address),
     },
     { condition: true, label: "model.route.fee", value: `${route.fee}%` },
-    { condition: true, label: "model.route.min_deposit", value: "0.1 DFI / 1 USD" },
+    {
+      condition: true,
+      label: "model.route.min_deposit",
+      value: `${route.minDeposit.dfi} DFI / ${route.minDeposit.usd} USD`,
+    },
```
Comment (davidleomay) [inline app/screens/home/RouteList.tsx:260]: Should we (for all min. deposits/invests) return a list of tuples with amount and asset in the API? Something like 
```ts
{
   ...
   minDeposit: [
      { amount: 0.1, asset: "DFI" },
      { amount: 1, asset: "USD" }
   ]
}
```
Would make it more generic.

Comment (Krysh90) [reply app/screens/home/RouteList.tsx:260]: yes felt a bit awkward to deliver it in different formats, tbh. Wasn't sure if we want to introduce something here.
I like that idea to adapt all to use the same MinDeposit interface. I will change it on API and then here.

File: app/screens/home/RouteList.tsx [hunk 2]
```diff
@@ -231,6 +238,11 @@ const RouteList = ({
       label: "model.route.fee",
       value: `${route.fee}%` + (route.refBonus ? ` (${route.refBonus}% ${t("model.route.ref_bonus")})` : ""),
     },
+    {
+      condition: true,
+      label: "model.route.min_deposit",
+      value: `${formatMinDeposit(route.minDeposits, "BTC")}`,
+    }, // BTC hardcoded until we get asset for deposit from API
```
Comment (davidleomay) [inline app/screens/home/RouteList.tsx:245]: I would solve this (for all routes) in a very generic way:
```ts
value: route.minDeposits.map((d) => `${d.amount} ${d.asset}`).join(' / '),
```

Comment (Krysh90) [reply app/screens/home/RouteList.tsx:245]: fair enough, but why should we show min deposits for other blockchains for example. If we have a min deposit for ETH. Then I wouldn't show it for other crypto-routes

Comment (davidleomay) [reply app/screens/home/RouteList.tsx:245]: IMO it's the task of the API to determine which min. deposits apply for the specific route.

**Review by davidleomay — APPROVED** [review]

**Pull request merged**